Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -63,21 +63,20 @@ public ApplicationContextManagerBlazor(IHttpContextAccessor hca, AuthenticationS
     /// </summary>
     protected HttpContext HttpContext { get; set; }
 
-    private void InitializeUser()
+    private async Task InitializeUser()
     {
       if (ActiveCircuitState.CircuitExists)
       {
         Task<AuthenticationState> task;
         try
         {
-          task = AuthenticationStateProvider.GetAuthenticationStateAsync();
+          await(task = AuthenticationStateProvider.GetAuthenticationStateAsync());
         }
         catch (InvalidOperationException ex)
         {
           task = Task.FromResult(new AuthenticationState(UnauthenticatedPrincipal));
           string message = ex.Message;
-          if (message.Contains(nameof(AuthenticationStateProvider.GetAuthenticationStateAsync))
-              && message.Contains(nameof(IHostEnvironmentAuthenticationStateProvider.SetAuthenticationState)))
+          if (message.Contains(nameof(AuthenticationStateProvider.GetAuthenticationStateAsync) + " outside of the DI scope for a Razor component"))
```

Comment:
When did the message get changed? We should `#if` or check for both versions of the exception to keep compatibility with older .Net versions.

Reply:
Good point.  Fixed for backward compatibility.